src/Microsoft.DocAsCode.Build.Engine/Microsoft.DocAsCode.Build.Engine.csproj

```diff
@@ -28,6 +28,7 @@
     <Compile Include="InvalidPreprocessorException.cs" />
     <Compile Include="CrossReferenceNotResolvedException.cs" />
     <Compile Include="DocumentBuildContext.cs" />
+    <Compile Include="MarkdownProviders\DfmJsonTokenTreeServiceProvider.cs" />
     <Compile Include="MarkdownProviders\JsonTokenTreeServiceProvider.cs" />
```

Comment:
But I think we should provide the JsontokenTreeServiceProvider .

---
In reply to: [77461877](https://github.com/dotnet/docfx/pull/667#discussion_r77461877) [](ancestors = 77461877,77453106)